tests/core/test_log.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import pytest
+
+from pytest_ethereum.testing import Log
+
+
+logging.getLogger("evm").setLevel(logging.INFO)
+
+
+@pytest.fixture
+def ping(vyper_project_dir, vy_deployer):
+    return vy_deployer.deploy("ping")
+
+
+def test_log_is_present(ping, w3):
+    ping_instance = ping.deployments.get_contract_instance("ping")
```

Comment:
have the `ping` fixture return `vy_deployer.deploy("ping").deployments.get_contract_instance("ping")`, then you don't need to repeat this line 3 times